integrations/DCMFloodlight/browser.js

```diff
@@ -22,6 +26,7 @@ class DCMFloodlight {
     this.allowAdPersonalizationSignals = config.allowAdPersonalizationSignals;
     this.doubleclickId = config.doubleclickId;
     this.googleNetworkId = config.googleNetworkId;
+    this.tagFormat = config.tagFormat;
```

Comment:
we need to add backward compatibility for any configuration changes. Anyone who is using the older config, will not have access to `tagFormat`, they need to be defaulted to one of the choices

Reply:
Can we use 
`this.tagFormat = config.tagFormat || globalSiteTag`

Reply:
added backward compatibility